lib/internal/Magento/Framework/DataObject.php

```diff
@@ -10,7 +10,6 @@
  *
  * @api
  * @SuppressWarnings(PHPMD.NumberOfChildren)
- * @since 100.0.2
```

Comment:
Could  you explain why you removed this line?

Reply:
It was mistake